Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -77,7 +78,11 @@ namespace Babylon::Polyfills::Internal
 
     Napi::Value NativeCanvas::GetContext(const Napi::CallbackInfo& info)
     {
-        return Context::CreateInstance(info.Env(), this);
+        if (m_contextObject.IsEmpty())
+        {
+            m_contextObject = Napi::Persistent(Context::CreateInstance(info.Env(), info.This()).As<Napi::Object>());
```

Comment:
Canvas holds context and context holds canvas. This is probably going to prevent the GC from collecting these objects once created. Can you check?

Reply:
With canvasObject persistent, context is not disposed nor Canvas when disposing the dynamic texture. When using `Napi::Weak` Canvas and Context are properly disposed when disposing dyn. texture.